code/_onclick/hud/screen_objects.dm

```diff
@@ -495,7 +495,7 @@
 		if("Track With Camera")
 			if(isAI(usr))
 				var/mob/living/silicon/ai/AI = usr
-				var/target_name = input(AI) in AI.trackable_mobs()
+				var/target_name = input(AI) as null|anything in AI.trackable_mobs()
 				AI.ai_camera_track(target_name)
```

Comment:
ai_camera_track itself is a verb that allows null arguments, in which case the tracking is specifically cancelled, so this is intentional.